include/event_manager.rst

```diff
@@ -311,10 +311,18 @@ Profiler allows you to observe the propagation of an event in the system, view t
 
 To profile an event, you must complete the following steps:
 
-1. Enable profiler with :option:`CONFIG_EVENT_MANAGER_PROFILER_ENABLED` Kconfig option.
+1. Enable the profiler using the :option:`CONFIG_EVENT_MANAGER_PROFILER_ENABLED` Kconfig option.
 #. Edit the source file for the event type:
 
-   a. Define a profiling function that logs the event data to a given buffer by calling :c:func:`profiler_log_encode_u32`, regardless of the profiled data type.
+   a. Define a profiling function that logs the event data to a given buffer by calling one of the following functions, depending on the profiled data type:
```

Comment:
one of the following functions for every registered data type

Reply:
fixed